src/accordion/accordion.tsx

```diff
@@ -0,0 +1,78 @@
+import { PropsWithChildren, useContext, useEffect, useMemo, useRef, useState } from 'react';
+import styles from './accordion.m.scss';
+import ArrowExpandDownSVG from '@sima-land/ui-quarks/icons/16x16/Stroked/ArrowExpandDown';
+import classnames from 'classnames/bind';
+import { AccordionContext } from './accordion-provider';
+
+const cx = classnames.bind(styles);
+
+interface Props extends PropsWithChildren {
+  title: string;
+  theme?: 'light' | 'dark';
+  description?: string;
+  groupId: string;
+  initialOpen?: boolean;
+}
+
+/**
+ * Аккордеон.
+ * @param props Пропсы компонента.
+ * @param props.title Заголовок каталога.
+ * @param props.description Описание.
+ * @param props.theme Тема оформления.
+ * @param props.initialOpen Первоначальное значение состояния открытия.
+ * @param props.children Контент аккордеона при открытии.
+ * @return Элемент.
+ */
+export const Accordion = ({ groupId, title, description, theme, initialOpen, children }: Props) => {
+  const container = useRef<HTMLDivElement>(null);
+  const { selected, register, unregister, toggle } = useContext(AccordionContext);
+  const [id, setId] = useState<symbol>(Symbol());
```

Comment:
```suggestion
  const [id] = useState<symbol>(() => register(groupId, initialOpen));
```
Чтобы не вызывать в хуке на монтирование.

Reply:
заменил, но без стрелочной функции

Reply:
Ну да, начальное значение установится разово. Но вот функция register будет вызываться на каждом рендере и, вероятно, вызывать какие то избыточные действия в провайдере. Оно же не нужно?

Reply:
точняк, поправил